src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -588,26 +588,25 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environmental and recording system's
-noise.
-In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
-each experimental session.
-It is RECOMMENDED to store the empty-room recording inside a subject directory
-named `sub-emptyroom`.
-The label for the [`task-<label>`](../99-appendices/09-entities.md#task) entity in the empty-room recording SHOULD be
-set to `noise`.
-If a [`session-<label>`](../99-appendices/09-entities.md#ses) entity is present, its label SHOULD be the date of the
-empty-room recording in the format `YYYYMMDD`, that is `ses-YYYYMMDD`.
-The `scans.tsv` file containing the date and time of the acquisition SHOULD
-also be included.
-The rationale is that this naming scheme will allow users to easily retrieve the
-empty-room recording that best matches a particular experimental session, based
-on date and time of the recording.
-It should be possible to query empty-room recordings just like usual subject
-recordings, hence all metadata sidecar files (such as the `channels.tsv`) file
-SHOULD be present as well.
+Empty-room MEG recordings capture the environmental and recording system's noise.
+In the context of BIDS it is RECOMMENDED to associate an empty-room recording to each experimental session (see `AssociatedEmptyRoom` field in the `*_meg.json` sidecar file).
```

Comment:
I rewrote this. But I'm not sure how to produce the correct link. Could sb please help?

```suggestion

It is RECOMMENDED to explicitly specify which empty-room recording should be used with which experimental run(s) or session(s). This can be specified via the [`AssociatedEmptyRoom`](99-appendices/14-glossary.html#associatedemptyroom-metadata) field in the `*_meg.json` sidecar files.
```

Reply:
You had it almost right, we just need to:

- add a "SPEC_ROOT/" in front of internal links
- use `.md` as extensions, instead of html

Reply:
ah no ... "SPEC_ROOT" was wrong, ... `../` was needed instead. We only need "SPEC_ROOT" when using internal links from within schema files.